docs/mocksupport.rst

```diff
@@ -878,3 +878,64 @@ DMTF CIM schema download support
       :attributes:
 
    .. rubric:: Details
+
+
+.. _`Mock inmemory CIM repository`:
+
+Mock inmemory CIM repository
+----------------------------
+
+pywbem_mock provides a standard interface to a CIM repository that
+stores CIM classes, CIM instances, and CIM qualifier declarations.
+
+This repository implements a repository in memory that is created and
+destroyed each time the FakedWBEMConnection class is constructed and
+destroyed.
```

Comment:
On "This repository implements a repository": This wording can be improved. Assuming my suggestion for the first sentence, I suggest to change this sentence here to something like:
```
Each :class:`~pywbem_mock.FakedWBEMConnection` object contains its own fresh
:class:`~pywbem_mock.InMemoryRepository` object that is the target of the mock
operations.
```

Reply:
CLOSED - became moot since the last commit changed the module docstring.